docs/_user_guide/julia-slurm.md

```diff
@@ -0,0 +1,59 @@
+# Running Julia on Yen-Slurm 
```

Comment:
When I edited the Slurm page, I changed all Yen-Slurm to Yen Slurm. Not sure which is better for consistency?